Scripts/script-ToTable.yml

```diff
@@ -5,7 +5,8 @@ commonfields:
 name: ToTable
 script: |-
   var headers = args.columns ? args.columns.split(',') : null;
-  return {Type: entryTypes.note, Contents: tableToMarkdown(args.title, args.data, headers), ContentsFormat: formats.markdown};
+  var flattenObject = treeToFlattenObject(args.data);
```

Comment:
What is use case of toTable, I never used it, it is better to understand when you are making fix :-)